pkg/scripts/download_test.go

```diff
@@ -30,8 +30,8 @@ func (f *FakeHTTPGet) get(url string) (*http.Response, error) {
 	}, f.err
 }
 
-func getHTTPReader() (*HttpURLReader, *FakeHTTPGet) {
-	sr := &HttpURLReader{}
+func getHTTPReader() (*HTTPURLReader, *FakeHTTPGet) {
```

Comment:
They were already "out of sync", see the red/old part of the diff:

```diff
-func getHTTPReader() (*HttpURLReader, *FakeHTTPGet) {
-	sr := &HttpURLReader{}
```

`getHTTPReader` returns `*HttpURLReader` (and also a `*FakeHTTPGet`?!)